connector_office_365/models/res_users.py

```diff
@@ -163,7 +169,7 @@ def office_365_request(self, method, url, data=None, headers=None):
                                    data=data,
                                    client_id=client_id,
                                    client_secret=client_secret)
-        if response.status_code not in [200, 204]:
+        if not response.ok:
```

Comment:
ok is just [200, 204] or it's wider ?

Reply:
Read the commit's message https://github.com/onesteinbv/addons-onestein/pull/78/commits/a61b58eba1f3b69d6af0765c45209b5436a3da9f

Yes, it is wider. It should cover all 2xx and 3xx codes.

Reply:
ok thanks :+1: